packages/patterns/package.json

```diff
@@ -18,7 +18,10 @@
   "main": "./index.js",
   "module": "./index.js",
   "exports": {
-    ".": "./index.js",
+    ".": {
+      "types": "./types.d.ts",
+      "default": "./index.js"
+    },
```

Comment:
I still prefer to have an `export * from './types.js'` in `index.js` (with `types.js` being a stub `export {}`)